### DISCUSSION TITLE
Is there a way to make any new note go to a specific path?

### DISCUSSION THREAD

**Discussion by aco-gt** [svsool/vscode-memo#620] [Q&A]
Is there a way to make any new note go to a specific path?
When I click [[new_note]], I want to make "new_note" into a specific path.

**Comment by svsool** [comment]
yes, there is a way, see [link rules](https://github.com/svsool/memo/blob/master/help/Features/Creating%20notes%20from%20links.md)

**Reply by aco-gt** [reply]
@svsool Thanks for the answer. That makes it much more useful. I think this plugin is great!
